theme.js

```diff
@@ -41,50 +50,39 @@ define(function(require, exports, module) {
         // when plugin is loaded
         plugin.on("load", function() {
 
-            // create button
-            button = new imports.ui.button({
-                "command": "toggleTheme",
-                "skin": "c9-menu-btn",
-                "tooltip": "Theme",
-                "visible": true
+            menuItem = new ui.item({
+                type: "check",
+                caption: "ToggleTheme",
+                onclick: toggleTheme
             });
 
+            menus.addItemByPath(
+                "View/PresentationDiv", new ui.divider(), 3, plugin
+            );
+            menus.addItemByPath("View/Night Mode", menuItem, 4, plugin);
+
             // register command for button
             imports.commands.addCommand({
                 exec: toggleTheme,
                 group: "CS50",
                 name: "toggleTheme"
             }, plugin);
 
-            // load CSS for button
-            imports.ui.insertCss(require("text!./style.css"), options.staticPrefix, plugin);
+            // get the current theme
+            setTheme();
 
-            // style button
-            styleButton();
-
-            // re-style button whenever theme changes
+            // reset global var whenever style changes
             imports.settings.on("user/general/@skin", function(value) {
```

Comment:
This could probably be just: `imports.settings.on("user/general/@skin", setTheme, plugin);`